tests/integration/adaptation-editor/custom-reporter.ts

```diff
@@ -0,0 +1,334 @@
+import { writeFile, mkdir } from 'fs/promises';
+import { join, basename } from 'path';
+import { existsSync } from 'fs';
+import type {
+    FullConfig,
+    FullResult,
+    Reporter,
+    Suite,
+    TestCase,
+    TestResult,
+    TestStep
+} from '@playwright/test/reporter';
+
+interface ManualTestCaseStep {
+    name: string;
+}
+interface ManualTestCase {
+    name: string;
+    filePath: string;
+    steps: ManualTestCaseStep[];
+}
+
+export default class ManualTestCaseReporter implements Reporter {
```

Comment:
Class name should match the file name - `manual-test-case-reporter.ts`.

Reply:
adapted